src/components/NavigationListItem.js

```diff
@@ -1,13 +1,11 @@
 import React from 'react';
-import ListGroupItem from 'react-bootstrap/ListGroupItem';
-import { LinkContainer } from 'react-router-bootstrap';
+import { ListItem, ListItemText } from '@material-ui/core';
```

Comment:
Let's start using the new convention from the Material-UI docs you pointed me to.

Reply:
There's a lot of these around, I'll create a new PR that has all of these updated.